tendrl/commons/message.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+from inspect import getframeinfo
+from inspect import stack
+import uuid
+
+
+class Message(object):
```

Comment:
Additionally, the `validate` method should check the attributes, like it currently does. It should not be validating json. Thus it must be called only after the attributes have been set.